### PR TITLE
Switch to GHA CI workflow

### PR DESCRIPTION
**Pull request by internalsystemerror**
Switch from Travis CI to use GitHub Actions according to: https://gist.github.com/weierophinney/b003e50c3c2667d08076caf31ebd36a4.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
 # Twig Integration for Mezzio
 
-[![Build Status](https://travis-ci.com/mezzio/mezzio-twigrenderer.svg?branch=master)](https://travis-ci.com/mezzio/mezzio-twigrenderer)
-[![Coverage Status](https://coveralls.io/repos/github/mezzio/mezzio-twigrenderer/badge.svg?branch=master)](https://coveralls.io/github/mezzio/mezzio-twigrenderer?branch=master)
+[![Build Status](https://github.com/mezzio/mezzio-twigrenderer/workflows/continuous-integration.yml/badge.svg)](https://github.com/mezzio/mezzio-twigrenderer/actions/workflows/continuous-integration.yml)
```
Comment (froschdesign) [inline README.md]: Unfortunately, the image path is wrong:
```suggestion
[![Build Status](https://github.com/mezzio/mezzio-twigrenderer/actions/workflows/continuous-integration.yml/badge.svg)](https://github.com/mezzio/mezzio-twigrenderer/actions/workflows/continuous-integration.yml)
```

Compare:

Build Status

Build Status

Comment (internalsystemerror) [reply README.md]: crap I copied my comment from the gist ... I'll have to correct this on quite a few, thanks

Comment (internalsystemerror) [reply README.md]: corrected on all packages

**Review by froschdesign — APPROVED** [review]

**Comment by froschdesign** [comment]
Thank you for your time and this contribution! 👍

**Comment by ElisDN** [comment]
Excuse me, but why was `composer.lock` indexed to Git?

**Comment by froschdesign** [comment]
@ElisDN 
The file is needed for the GitHub action and the related checks – see the tab "[Checks](https://github.com/mezzio/mezzio-twigrenderer/pull/7/checks)" on each pull request.

https://github.com/marketplace/actions/laminas-continuous-integration

**Pull request merged**